dist/profile/templates/buildmaster/casc/clouds.yaml.erb

```diff
@@ -76,13 +76,13 @@ jenkins:
   <%- @cloud_agents["ec2"].each do |ec2_cloud_name, ec2_cloud_setup| -%>
   - amazonEC2:
       cloudName: <%= ec2_cloud_name %>
-      credentialsId: "aws-credentials"
+      credentialsId: "<%= ec2_cloud_setup["credentialsId"] %>"
       <%- $maxEC2Vms = 0; ec2_cloud_setup["agent_definitions"].each do |agent| -%>
         <%- $maxEC2Vms = agent["maxInstances"] + $maxEC2Vms -%>
       <%- end -%>
       instanceCapStr: "<%= $maxEC2Vms %>"
       region: "<%= ec2_cloud_setup["region"] %>"
-      sshKeysCredentialsId: "ec2-agent-ssh-2021-06"
+      sshKeysCredentialsId: "<%= ec2_cloud_setup["sshKeysCredentialsId"] %>"
```

Comment:
questionable if this needs to be configurable

Reply:
Same (the credential is different between ci.j, trusted.ci and others (both its id and the value), hence the need to configure it.)